core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/random.h

```diff
@@ -14,8 +14,10 @@
 #ifndef RANDOM_H
 #define RANDOM_H
 
-void
+__wasi_errno_t
```

Comment:
Should there also be an `include` directive for this type? Otherwise you'll depend on the order of including the file, which usually isn't a good practice.

Reply:
thanks for spotting, added an include for `bh_platform.h`